src/content/en/2022/security.md

```diff
@@ -1,13 +1,15 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: Security
-#TODO - Review and update chapter description
 description: Security chapter of the 2022 Web Almanac covering Transport Layer Security, content inclusion (CSP, Feature Policy, SRI), web defense mechanisms (tackling XSS, XS-Leaks), and drivers of security mechanism adoptions.
 authors: [SaptakS, feross, lirantal]
 reviewers: [kushaldas, tomvangoethem, nrllh, clarkio]
 analysts: [VictorLeP, vikvanderlinden, GJFR]
-editors: []
+editors: [tunetheweb]
 translators: []
+SaptakS_bio: Saptak S is a human rights centered web developer, focusing on usability, security, privacy and accessibility topics in web development. He is a contributor and maintainer of various different open source projects like <a hreflang="en" href="https://www.a11yproject.com">The A11Y Project</a>, <a hreflang="en" href="https://onionshare.org/">OnionShare</a> and <a hreflang="en" href="https://wagtail.org/">Wagtail</a>. You can find him blogging at <a hreflang="en" href="https://saptaks.blog">saptaks.blog</a>.
+feross_bio: Feross Aboukhadijeh, is an entrepreneur, programmer, open source author, and mad scientist. He maintain <a hreflang="en" href="https://www.npmjs.com/~feross">100+ packages on npm</a> which are downloaded 500+ million times per month.
+lirantal_bio: Liran is <a hreflang="en" href="https://stars.github.com/profiles/lirantal/">a GitHub Star</a>, world-wide recognized for championing open source software and actively working within communities to inspire and lift other humans. A <a hreflang="en" href="https://github.com/nodejs/security-wg">JavaScript & Node.js software developer, building web applications and command-line tools</a>. <a hreflang="en" href="https://owasp.org/www-project-cwe-toolkit/">A web security activist</a>, engaging in security research, software supply chain security, and regular contributor and project lead to OWASP Foundation projects. <a hreflang="en" href="https://github.com/nodejs/security-wg">An avid member of the Node.js Foundation ecosystem security working group</a>, dedicated to advancing Node.js security awareness and skillset in the open source community. <a hreflang="en" href="https://snyk.io/">Developer Advocate at Snyk</a>.
```

Comment:
@lirantal I stole this from your website but please change as you see fit.

Reply:
Thanks, revised here:

```suggestion
lirantal_bio: Known for his open source and JavaScript security initiatives, <a hreflang="en" href="https://www.lirantal.com/">Liran Tal</a> is an award-winning software developer, security researcher, and open source champion in the JavaScript community. He's an internationally recognized <a hreflang="en" href="https://stars.github.com/profiles/lirantal/">GitHub Star</a>, acknowledged for his open source advocacy, and has received the OpenJS Foundation's Pathfinder for Security for his work on Node.js security. His contributions to developer security education include leading OWASP projects, building supply chain security tools, participation in CNCF and OpenSSF initiatives, and authoring books such as O'Reilly's Serverless Security. He leads the developer advocacy team at Snyk.io and is on a mission to empower developers with better application security skills.
```

Reply:
also noting that @clarkio should add his bio as an author too.